be/src/starrocks_format/starrocks_lib.cpp

```diff
@@ -0,0 +1,103 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
```

Comment:
license header declamation should be fixed.

Reply:
fixed